classes/massactionutils.php

```diff
@@ -62,16 +62,18 @@ public static function extract_modules_from_json(string $massactionrequest): std
             'ID IN (' . implode(',', array_fill(0, count($data->moduleIds), '?')) . ')',
             $data->moduleIds);
 
+        $orderedmodules = array();
```

Comment:
```suggestion
        $orderedmodules = [];
```
just to keep it consistent :-)